tests/acceptance/variant_browser.js

```diff
@@ -289,50 +295,58 @@ function positionFilterBoxInValidation(driver){
-            driver.findElement(By.xpath("//div[contains(@class,'x-window x-message-box')]//span[contains(@class,'x-btn-inner x-btn-inner-default-small')]")).click();
-        });
-    });
+    assertAlertWindowShown(driver, 'Please enter the correct range.The start of the region should be smaller than the end');
 
     driver.findElement(By.name("region")).clear();
     driver.findElement(By.name("region")).sendKeys('1:3000000-310000000');
     driver.findElement(By.id("vb-submit-button")).click();
-    config.sleep(driver);
-    driver.wait(until.elementLocated(By.xpath("//div[contains(@class,'x-window x-message-box')]//div[contains(@class,'x-component x-window-text x-box-item x-component-default')]")), config.wait()).then(function(text) {
-        driver.findElement(By.xpath("//div[contains(@class,'x-window x-message-box')]//div[contains(@class,'x-component x-window-text x-box-item x-component-default')]")).getText().then(function(text){
-            assert(text).equalTo('Please enter a region no larger than 1 million bases');
-            driver.findElement(By.xpath("//div[contains(@class,'x-window x-message-box')]//span[contains(@class,'x-btn-inner x-btn-inner-default-small')]")).click();
-        });
-    });
+    assertAlertWindowShown(driver, 'Please enter a region no larger than 1 million bases');
 
     driver.findElement(By.name("region")).clear();
     driver.findElement(By.name("region")).sendKeys('1,13:12233-12234');
     driver.findElement(By.id("vb-submit-button")).click();
+    assertAlertWindowShown(driver, 'Please enter a valid region');
+
+    driver.findElement (By.xpath ("//div[contains(@id,'VariantWidgetPanel')]//span[text()='Reset']")).click ();
+}
+
+function assertAlertWindowShown(driver, message) {
     config.sleep(driver);
-    driver.wait(until.elementLocated(By.xpath("//div[contains(@class,'x-window x-message-box')]//div[contains(@class,'x-component x-window-text x-box-item x-component-default')]")), config.wait()).then(function(text) {
-        driver.findElement(By.xpath("//div[contains(@class,'x-window x-message-box')]//div[contains(@class,'x-component x-window-text x-box-item x-component-default')]")).getText().then(function(text){
-            assert(text).equalTo('Please enter a valid region');
+    driver.wait(until.elementLocated(By.xpath("//div[contains(@class,'x-window x-message-box')]//div[contains(@class,'x-component x-window-text x-box-item x-component-default')]")), config.wait()).then(function (text) {
+        driver.findElement(By.xpath("//div[contains(@class,'x-window x-message-box')]//div[contains(@class,'x-component x-window-text x-box-item x-component-default')]")).getText().then(function (text) {
+            assert(text).equalTo(message);
             driver.findElement(By.xpath("//div[contains(@class,'x-window x-message-box')]//span[contains(@class,'x-btn-inner x-btn-inner-default-small')]")).click();
         });
     });
+}
 
+function emtpyPositionFilter(driver){
```

Comment:
This method should be split in 3 separate test cases. Also `empty` instead of `emtpy`.